lib/dialects/mssql/schema/mssql-columncompiler.js

```diff
@@ -63,6 +64,22 @@ class ColumnCompiler_MSSQL extends ColumnCompiler {
     return '';
   }
 
+  defaultTo(value, { constraintName }) {
```

Comment:
This needs to be slightly more defensive about argument not being passed, so we probably can't destructure it always

Reply:
This is the problem with making changes at midnight!

I've updated this to provide a default value for the argument; is that acceptable? I've also updated the typings

Reply:
yup, this looks good! Do you need any help with writing a test?